lib/ActivityStreams.js

```diff
@@ -98,7 +98,8 @@ function ActivityStreams(options = {}) {
   );
 
   this._metadataStore = new MetadataStore();
-  this._previewProvider = new PreviewProvider(this._tabTracker);
+  this._metadataStore.asyncConnect();
```

Comment:
@sarracini @ncloudioj should we be doing this here, or should this happen inside the MetadataStore init?